code/datums/diseases/advance/symptoms/narcolepsy.dm

```diff
@@ -63,7 +63,7 @@ Bonus
 		if(5)
 			if(prob(50))
 				to_chat(M, "<span class='warning'>[pick("So tired...","You feel very sleepy.","You have a hard time keeping your eyes open.","You try to stay awake.")]</span>")
-			M.drowsyness = CLAMP(M.drowsyness, M.drowsyness + 40, 70)
+			M.drowsyness = min(M.drowsyness + 40, 70)
```

Comment:
```suggestion
			min(M.drowsyness + 40, max(70, M.drowsyness))
```

Reply:
Or
```suggestion
		CLAMP(70, M.drowsyness + 40, M.drowsyness)
```
should work too if I'm not completely demented

Reply:
Wait, did I misunderstand how CLAMP works? I thought it just chose the non-highest, non-lowest value of the three values in the CLAMP, without caring about their ordering.

Reply:
oh god i did oh shit oh fuck hold on i need to make a big fix PR this is bad oh god